src/snapshot.ts

```diff
@@ -13,6 +13,20 @@ function genId(): number {
   return _id++;
 }
 
+function getValidTagName(tagName: string): string {
```

Comment:
I'm wondering whether this may transform to some built-in tag with specific UI, like:
- a1 -> a
- b1 -> b
- link' -> link

Is fallback all of this to div a better idea or not?

Such as:
```js
try {
  createElement(tagname)
} catch {
  tagname = 'div'
}
```

Reply:
That's a good idea. We can try creating the element and on fail return div. Im wondering whats the overhead on creating an element every time. Perhaps we should just test to see if the tagName has a number or symbol, then just return div if so